composer/src/main/kotlin/com/gojuno/composer/Main.kt

```diff
@@ -39,6 +39,9 @@ fun main(rawArgs: Array<String>) {
 
     val suites: List<Suite> = connectedAdbDevices()
             .map {
+                it.filter { args.devices.isEmpty() || args.devices.contains(it.id) }
```

Comment:
Maybe something like 

```kotlin
when (args.devices.isEmpty()) {
  true -> it
  false -> it.filter { args.devices.contains(it.id) }
} 
```

will be more clear in its intention in the long term? 

Because I caught myself second time trying to figure out why you do `args.devices.isEmpty()` check first in the `filter {}` :)